components/python/psycopg2/psycopg2-PYVER.p5m

```diff
@@ -37,12 +46,5 @@ file path=usr/lib/python$(PYVER)/vendor-packages/psycopg2/extras.py
 file path=usr/lib/python$(PYVER)/vendor-packages/psycopg2/pool.py
 file path=usr/lib/python$(PYVER)/vendor-packages/psycopg2/sql.py
 file path=usr/lib/python$(PYVER)/vendor-packages/psycopg2/tz.py
-#file path=usr/lib/python$(PYVER)/vendor-packages/psycopg2/_psycopg.so
-
-# force a dependency on the Python $(PYVER) runtime
-depend type=require fmri=__TBD pkg.debug.depend.file=python$(PYVER) \
-    pkg.debug.depend.path=usr/bin
-
-# force a dependency on the psycopg2 package
-depend type=require \
-    fmri=$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+#file path=usr/lib/python$(PYVER)/vendor-packages/psycopg2/_psycopg.cpython-39.so
```

Comment:
>> Is this (and |_psycopg.cpython-37m.so| for 3.7) really not needed to make the package working correctly? If so, why is it built?
>
> Github seems to be confused. I cannot see your last messages in the browser but only got emails. So I miss the context and can only guess. I have commented these two lines and manually added the last line because we have transforms/python-3-soabi which manipulates suffixes of .so files.

Ok, thanks.  Makes sense, I think.